docs/memory.md

```diff
@@ -0,0 +1,104 @@
+---
+title: Using the Memory View
+---
+
+* toc
+{:toc}
+
+## What is it?
+
+Dynamically allocated Dart objects e.g., ```new ClassName()``` live in a portion of memory called the heap.
+
+DevTool’s Memory profile tab lets you peek at how an isolate is using memory at a given moment. This page also provides accumulator counts that you can use to study the rate of memory allocation. The accumulators are useful if you suspect that your application is leaking memory, or has other bugs relating to memory allocation.
```

Comment:
Should we call this a "tab" or a "screen"? Whichever way we choose we should normalize all our documentation and error messages to match that option. I know we have at least some cases where we call the pages "screens" in our error messages.

Reply:
I looked it up in the Google Developers Doc Guide and it should be called a tab:
https://developers.google.com/style/ui-elements#term-tab

Reply:
TAB.  :)

Reply:
Thanks changed from 'Memory profile tab' to 'Memory tab'.

Reply:
However, what should we call the area that appears when a navigation Tab is clicked on?
We seem to use 'screen' which I don't like.  I prefer pane, see [Google Developers Doc Guide](https://developers.google.com/style/ui-elements#term-windows)

Any suggestions?